easypost/requestor.py

```diff
@@ -125,6 +130,7 @@ def requests_request(self, method, abs_url, headers, params):
         return http_body, http_status
 
     def urlfetch_request(self, method, abs_url, headers, params):
+        """Make a request by using the `urlfetch` library from Google app engine."""
```

Comment:
Keep it generic when possible. Maybe we add or subtract from this.

```suggestion
        """Make a request by using the `urlfetch` library."""
```